video/out/gpu/d3d11_helpers.h

```diff
@@ -72,6 +78,8 @@ struct d3d11_swapchain_opts {
     int width;
     int height;
     DXGI_FORMAT format;
+    DXGI_COLOR_SPACE_TYPE color_space;
+    struct mp_colorspace *configured_csp;
```

Comment:
This should probably have a comment, since it's unusual to have an output parameter here. It should probably mention it's okay for configured_csp to be NULL, as well.

Reply:
Comment has been added. Also this interface can be changed now or in the future if it is so desired.